default/python/AI/PriorityAI.py

```diff
@@ -161,6 +162,11 @@ def _calculate_research_priority():  # noqa: max-complexity
     galaxy_is_sparse = ColonisationAI.galaxy_is_sparse()
     enemies_sighted = aistate.misc.get("enemies_sighted", {})
 
+    # get capital's preferred focus
```

Comment:
Instead of comments that just duplicates the code behaviour, lets rework it in one of the following ways: 

- Create a function that returns capital preferred focus.   I like this approach because it does not add temporary variables to the current function namespace.
- Rename capital_focus to capital_preffered_focus

Also, let's move the variable declaration close to its usage.   Declare it here and use 20 lines later making code harder to read.